app/helpers/repository_datatable_helper.rb

```diff
@@ -67,7 +87,7 @@ def prepare_simple_view_row_columns(repository_rows)
         row['2'] = {
           stock_present: record.repository_stock_cell.present?,
           value: {
-            consumed_stock_formatted: record.consumed_stock
```

Comment:
Why did you change this?

".select('SUM(my_module_repository_rows.stock_consumption) AS "consumed_stock"')"